src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -48,13 +47,12 @@ protected AbstractFindUsagesService(IThreadingContext threadingContext)
 
             await context.SetSearchTitleAsync(
                 string.Format(EditorFeaturesResources._0_implementations,
-                FindUsagesHelpers.GetDisplayName(tuple.Value.symbol))).ConfigureAwait(false);
+                FindUsagesHelpers.GetDisplayName(symbolAndProjectId.Symbol))).ConfigureAwait(false);
```

Comment:
note: i have an upcoming PR which removes all the symbolAndProjectId goop. And it makes so much of this so much nicer :)